### DISCUSSION TITLE
PROJECT: Public Portal

### DISCUSSION THREAD

**Discussion by magpiedin** [fieldmuseum/dams-netx#27] [General]
## Public Portal 
(formerly 'Test Photo Archives (PA) portal')

### Goals:
- clarify features that PA users as well as in-house & public users needed in PA NetX portal
- make publicly accessible portal for public & in-house users to find PA media and submit download requests

### Dependencies:
  - DNG workflow -- TODO: add workflow to the repo

### Stakeholders / Contacts:
- Photo Archives
- download-requesters

### When / Calendar: 
- Deliver: End of July

### What / Deliverables
- Something along these lines: https://dams-test.fieldmuseum.tech/portals/photo-archives/

### Related Resources
- Zendesk Ticket: [71623](https://fieldmuseum.zendesk.com/agent/tickets/71623)
- EMu Event irn 10110

**Comment by rondlg** [comment]
@magpiedin 

Some thoughts (sorry they are late)...

Reviewing this version: https://dams-test.fieldmuseum.tech/portals/museum-media/#page/home

### Tabs
- Taking a lesson from the Fuzzy-folks. The tab links should stay in NetX.
- - Field Museum: So since the logo goes to fm home we shouldn't need to repeat it in the menu.
- - About: Let's put this in the text blurb at the top then we can explain that it goes to the github repo.

### Homepage
- How do we feel about putting the master norms and citations information on the NetX side and then linking to them from FieldMain instead of the way we have it now? Can you have informational pages in NetX?" Then we could have a tab called "Permissions" that stayed in the NetX environment.
- - Do we need/Should we have "What does that mean in a Field Museum context?" faqs. For example in human-ese what does CC-BY-NC mean to me?

### Individual Asset Page(s)
- We have the mm uri in the citation field, what if there was a way to include a link to the matching CC page/faq in the rights summary? - - - Do we/Should we add that to the rights record in EMu so it can pull through?

**Reply by magpiedin** [reply]
Many thanks

### re:Tabs
- FM-logo in upper-left: this unfortunately only links to the portal-home (not www.fieldmuseum.org) -- if we can't adjust that, do we want to keep the 'Field Museum' text/tab link?
- About -- now updated [on the portal homepage](https://dams-test.fieldmuseum.tech/portals/museum-media/#page/home) & blurb is editable [here](https://dams-test.fieldmuseum.tech/app/#system/portal/101/pages/101) 

### re:Homepage
- We could do that if you think it's appropriate. One maybe-caveat: the NetX portal pages don't seem to have an audit trail to track changes to the portal-page content/html
- FAQs / human-ese sounds helpful -- will round up some of what we have so far

### re:Individual Asset Page(s)
- Good point -- The URLs should be in the Rights records (we added that field a few dev-rounds back), so I'll add it to what pulls through from EMu to NetX.

**Comment by rondlg** [comment]
### re:Tabs
FM-logo in upper-left: If we can't link to an external url, then leave it without one and link to fm.org in the text.

### re:Homepage
"...putting the master norms and citations information on the NetX side..."
- Don't mind either way, was just a suggestion, so up to you which you think is best.

**Comment by rondlg** [comment]
Portal v1 screenshots:
- HOME PAGE: [screencapture-dams-test-fieldmuseum-tech-portals-photo-archives-2024-07-23-10_19_26](https://github.com/user-attachments/assets/862128ba-7e62-4510-a0ee-09151a7997bf)
- MEDIA ASSETS TAB: [screencapture-dams-test-fieldmuseum-tech-portals-photo-archives-2024-07-23-10_19_59](https://github.com/user-attachments/assets/1cb55708-f388-42f0-8610-2401ac5d4cc0)
- BROWSE MUSEUM MEDIA TAB: [screencapture-dams-test-fieldmuseum-tech-portals-photo-archives-2024-07-23-10_20_21](https://github.com/user-attachments/assets/df725014-064f-4fe0-a7f0-93a155c11cd8)
- FIELD MUSEUM PHOTO ARCHIVES TAB:  [screencapture-fieldmuseum-org-department-photo-archives-2024-07-23-10_20_39](https://github.com/user-attachments/assets/856cb685-c9ae-479d-afdc-061c106629bf)

**Comment by RebeccaWilke** [comment]
I add more comments/suggestions to a document. Since NetX (external site) is a work in progress some of the comments might be a work in progress, that I am not aware of or might not be able to implement. 

[Document](https://docs.google.com/document/d/1tCWWbSDm-tH8sTJJB8tpcknWma3KjfnayeClQC8oKYc/edit?usp=sharing)

**Comment by magpiedin** [comment]
Many thanks @RebeccaWilke -- responses below (and noted where still in the works): 
(_& if it helps, we can split these out to separate discussions. For now:_)

**Q-1: Does the title of Group in EMu or the Collection in NetX affect the title of the Feature Media group/collection? If it does I might want to change the title of the EMu Groups I made and shared with you.**

A: The group title doesn't need to be displayed on the Featured Media page.  
Instructions on what to include when contacting helpdesk to add a Featured Media group are now [added here](https://docs.google.com/document/d/1K5QC_7d8A6FzL6qWDsCphrhHKkiMbyuiKJPAToowTJY/edit#heading=h.leshbydmtxka). Does that work?
(As noted [there](https://docs.google.com/document/d/1K5QC_7d8A6FzL6qWDsCphrhHKkiMbyuiKJPAToowTJY/edit#heading=h.o96eg734zvlc): NetX collection names are currently how we link/match to the corresponding EMu group, so those should stay as is. And it's ok/good to specify a different title for public display.)

**Q-2. What happens to Feature Media groups if they are created in EMu or NetX (internal site) by staff and if they decide to leave the museum? Once a person leaves the museum do their collections or groups connected to NetX (external site) disappear? (checking permanence of the group/work)**

A:  This relates to user management in EMu.  Periodically, EMu & NetX users for individuals who have left are cleared out upon request from their department/s.  In EMu (& subsequently NetX), their groups are transferable to another current user if specified.

**Q-3: I know you mentioned there are limitations on how a group/collection is displayed on the Feature Media section, is that an Axiell setting, or budget issue, or both?**

A: The Featured Media page is an out-of-the-box NetX (software) setting for how collections currently display in a NetX portal, but we may be able to set it up differently on our side.  e.g.:
1. original out-of-the-box collection display: https://dams.fieldmuseum.tech/portals/museum-media/#page/featured-media
2. alternate replacement collection page: https://dams.fieldmuseum.tech/portals/museum-media/#page/alt-featured-media
    - I'd recommend we switch to this 2nd page-layout where we could adjust some of the displayed info if needed. 

If you have specific layout/display-requests, could you add a new 'featured media layout' github issue or discussion?


**Q-4: In a previous meeting, you mentioned conducting a search on the NetX site and using filters to get the desired results, which can be grouped on the Feature Media page to help with formatting the page. What happens if you cannot remove the undesired media in search results for the group on the Feature Media page?**

A:  Setting up a static group (collection) instead of a saved search may be best.  Do you have a specific example? (We could workshop this at the next Sept 3 NetX user group call)

**Q-5: Also when creating a group/collection for the Feature Media page are there any minimums or limits on the number of media assets with a group, and also are/should there be limits for the number of Feature Media groups?**

A: Technically, no minimum/maximum, and the [suggested alternate layout](https://dams.fieldmuseum.tech/portals/museum-media/#page/alt-featured-media) would likely be better at handling variable collection-sizes.  
If it also helps to clarify the purpose of Featured Media, the intention is to mirror the 'Featured Projects' section from the collections sites -- e.g. near the top of https://collections-botany.fieldmuseum.org 

**Q-6: Need to confirm how photos show up in the Related Events filter (expeditions photos, exhibits photos, Women's board meetings, preview for press exhibit, etc.) on NetX (external site). Does an Event record in EMu need to be created, and are the media assets related to that event attached in the Multimedia tab of the Event record?**

A: Yes -- Events to which a Multimedia record is directly attached will show in the 'Related Events' field.  
([DAMS/EMu Mapping here](https://docs.google.com/spreadsheets/d/1idSTTGCJQvDgowKP4_ijFtcC_9GJGlzmM1JyzJjCxEg/edit?gid=0#gid=0&range=A23))

**Q-7: How do you keep Event records for internal use and/or external use? Does the Event record need a "yes" for internet or other setting changes for it to be on NetX (external site)? Would I need to write a HelpDesk Ticket to move the Event record to be on NetX (external site)? If so I would like to have "William V. Kelley - Roosevelts Eastern Asia Expedition" (IRN: 6226) appear in the Related Events filter.**

A: The Kelley-Roosevelts Event irn 6226 Multimedia should be updated/searchable by Tuesday afternoon. There appeared to be a problem in EMu with the attachment.  If you have problems with other Event-related data not migrating to NetX, could you add the Event IRN/s in a helpdesk ticket?

Normally: If a Multimedia record is directly attached to an Event:
- the Event Number, Summary Data, and Event Type for that Event are copied to the NetX asset's Related Events field
- the Event's GUID is copied to the NetX asset's Related Event URL (e.g. "https://pj.fieldmuseum.org/event/44d0e355-6344-40cd-946f-a52d950f3b86")
- There isn't a separate table in NetX for Events, and no other need for helpdesk ticket.

**Q-8: I know in the past people were told to put a certain amount of photos in the Multimedia tab and if a person needed to link over 100 photos to the Event record they were told to create a group and attach it to the Event record, has that changed?**

A: Currently no change.  EMu v9 updates should allow for larger attachments -- we will keep you posted.

**Q-9: Is there a limit on the number of Related Events that appear in the filter?**

A: Not as far as I know.
* _update 2024-aug-29_ -- For any filter (Related Events, Subjects, etc), the 25 most commonly used values are listed in alphabetical order

**Q-10: I want to confirm the results in filters are arranged alphabetically. And if not, then how are they arranged?**

A: The results within a given filter are arranged alphabetically (e.g. within Subjects, "Asia", "branding", etc)
The list of filter-fields itself ("Subjects", "Rights Summary", etc) is not alphabetical currently, but we can re-order it if needed. 
If you'd prefer the fields to be listed in a different order, could you specify/propose it in a new 'public portal - filter order' github issue?

**Q-11: If there are limitations on the Featured Media page, would the Subject filter be another alternative or tool to group results? Also, is there a standard for terms appearing in the Subject field? Are we supposed to use the Library of Congress Subject Headings (LCSH)? (If we are, I do not think the Subject terms are LCSH compliant.)**

A:  Data cleanup in the Subject field is always worth a try with the EMu user group (a note is added in the [EMu QA doc here](https://docs.google.com/document/d/1Uw6jIsLXJJW4oo9EApcqv5Wzihr-bhxyXAjKV4TtXlg/edit#heading=h.6u4w2wew9cob)) 

**Q-12: I noticed in the Subject filter there 210 results for “Crane Pacific Expedition” and in the Related Events “PA118.1 | Expedition/Field Trip | Crane Pacific Expedition ” there are 208 results. Is that a clerical error on someone’s part? Will we have to ask who added the Subject term(s) and grouped the photos for the Related Events filter for any updates?**

A: To check:
- Search in EMu Multimedia for Subjects = X
- Search in EMu Events for Event Title = Y, and see the directly attached Multimedia
If you see a discrepancy there and need help cleaning / attaching records, contact the record owners &/or helpdesk


**Q-13 -- Dump questions into a comment on the GitHub section for creating Feature Media (https://github.com/fieldmuseum/dams-netx/issues/37)?**
Yes -- going forward, additional feedback on the Featured Media page can go in https://github.com/fieldmuseum/dams-netx/issues/37
(A few questions above might benefit from separate new issues, but ok to start in #37 )
